activation/validation.go

```diff
@@ -389,6 +389,10 @@ func (v *Validator) verifyChainWithOpts(
 	if err != nil {
 		return fmt.Errorf("get atx: %w", err)
 	}
+	if atx.Golden() {
+		log.Debug("not verifying ATX chain", zap.Stringer("atx_id", id), zap.String("reason", "golden"))
```

Comment:
NIT: Its not that you don't verify the ATX chain. A golden ATX is valid by definition, so you just found an ATX that is already validated 🙂